### ISSUE TITLE
[Feature Request]: Open a Patreon account to help fix CodeLite

### ISSUE THREAD

**Issue by yunkot** [eranif/codelite#3494]
### Describe your idea in details

It looks like there is a lot of CodeLite development going on, whereas many obvious bugs (e.g. #3047,  #3052, #3231, #3315 and so on) are not getting addressed, some for years already. 

I use CodeLite for full-time work, and am always scared to upgrade (doing it once 1-2 years now) only to encounter new regressions in code completion and basic editing, and I don't use any of the new plugins or features. 

If you would consider opening a Patreon account or similar, I would gladly provide some regular small financial aid to help with CodeLite work and I'm sure there will be others who can do the same, which could help to start fixing bugs.

**Comment by Jarod42** [comment]
There is still https://codelite.org/support.php for donation

**Comment by yunkot** [comment]
Yes, but that is one time payment only. Something like GitHub Sponsors or Patreon enables donations on regular basis, e.g. monthly. Furthermore, it is more than just donations as it enables some extras to those who provide regular financial support. 

Some projects using this system are receiving substantial financial income, which enables developers to focus on the products full-time and even enables hiring additional team members to help with the work. I think CodeLite project has a lot of potential for this.

**Comment by yunkot** [comment]
I would also add #3459 and #3505 to the list of "obvious" UI/usability bugs.

**Comment by AJenbo** [comment]
I would add https://github.com/eranif/codelite/issues/3378
